cql/fhir-cql/src/main/java/com/ibm/fhir/cql/engine/searchparam/SearchParameterResolver.java

```diff
@@ -41,7 +41,7 @@ public SearchParameter getSearchParameterDefinition(String resourceType, String
         Map<String, SearchParameter> params = SearchUtil.getSearchParameters(resourceType);
 
         for (SearchParameter param : params.values()) {
-            if (name != null && param.getName().getValue().equals(name)) {
+            if (name != null && param.getCode().getValue().equals(name)) {
```

Comment:
This is a good find. I went ahead and checked the call hierarchy for getCode and the above are the only situations where it is used.